warehouse/models/mart/payments/_payments.yml

```diff
@@ -119,3 +119,54 @@ models:
       - name: transaction_date_pacific
       - name: day_of_week
         description: Day of week value of the transaction in the range [1,7] with Sunday as the first day of the week.
+  - name: fct_elavon__transactions
```

Comment:
Question, maybe for the larger Payments group -- do we want vendor names in these table names? I have historically tried not to do that, but not sure if it's necessary/appropriate for Payments.